server.py

```diff
@@ -57,11 +62,13 @@ def do_select_iteration(self):
 
 
 class Asyncable(object):
+    """TODO: Docstring."""
+
     def readable(self):
-        """Should I sign up to accept incoming data?"""
+        """Should I sign up to accept incoming data?."""
```

Comment:
PEP-257 requires the summary to end with a `.`. To avoid the warnings I've restructured the sentence now so it refers to the question but is no longer a question itself.